### PR TITLE
Storybook components vol. 2

### PR DESCRIPTION
**Pull request by AndreyGorelik**

### CODE REVIEW

File: src/shared/ui/userpicGallery/userPicGallery.styles.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  row: {
+    flexDirection: 'row',
+  },
+  userpic: {
+    borderRadius: 100,
+    borderWidth: 3,
+    borderColor: 'white',
```
Comment (HenoL-rgb) [inline src/shared/ui/userpicGallery/userPicGallery.styles.ts:10]: color

**Review by HenoL-rgb — APPROVED** [review]

File: src/shared/ui/postFullHost/postFullHost.types.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  id: string;
+  name: string;
+  photo: ImageSourcePropType;
+  phone?: string;
+  telegram?: string;
+}
+
+export interface PostFullHostProps {
+  photo: ImageSourcePropType;
+  post_tags: PostTag[];
```
Comment (PetrifiedEye) [inline src/shared/ui/postFullHost/postFullHost.types.ts]: Suggest to change names here

File: src/shared/ui/selectList/selectList.styles.ts
Comment (PetrifiedEye) [inline src/shared/ui/selectList/selectList.styles.ts:1]: There's no styles, thus it's better to remove that file

File: src/utils/convertTime/convertTime.ts
Comment (PetrifiedEye) [inline src/utils/convertTime/convertTime.ts:1]: Same here

**Review by PetrifiedEye — CHANGES_REQUESTED** [review]

**Review by PetrifiedEye — APPROVED** [review]

**Pull request merged**